### PR TITLE
Add simple configuration guides for all supported OpenStack services

### PR DESCRIPTION
**Pull request by janhorstmann**
Part of https://github.com/osism/issues/issues/938

### CODE REVIEW

**Comment by janhorstmann** [comment]
I have taken the list of supported services from the [release respository version 7.0.0](https://github.com/osism/release/blob/main/7.0.0/openstack-2023.2.yml#L37)

Do you want to keep the links currently used in the `nova` docs or want it to be more uniform?

The `skyline-apiserver` project seems not to have an admin guide, so I have not added a link

**Comment by berendt** [comment]
Use a single skyline file and add console & api server there.

Add additional neutron services (dynamic routing, vpnaas) to the neutron file and not in separate files.

**Comment by berendt** [comment]
Always add a link (if it exists) to the configuration reference (like https://docs.openstack.org/nova/latest/configuration/config.html) as well.

**Comment by berendt** [comment]
For some services the configuration guide is the same like the configuration reference. In this case only add a single link and name it configuration reference.

The configuration reference is the page of all usable configuration parameters.

The configuration guide explains how to configure specific features.

**Review by berendt — CHANGES_REQUESTED** [review]
As commented before.

**Comment by janhorstmann** [comment]
> For some services the configuration guide is the same like the configuration reference. In this case only add a single link and name it configuration reference.
> 
> The configuration reference is the page of all usable configuration parameters.
> 
> The configuration guide explains how to configure specific features.

Some services have sites titled "configuration **guide**", which only link to the configuration reference of one or more applications. Ich have included those as "configuration reference"  now, because they do not provide any further guidance, but the mismatch might be irritating for some.

File: docs/guides/configuration-guide/openstack/octavia.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+sidebar_label: Octavia
+---
+
+# Octavia
+
+* [Octavia admin guide](https://docs.openstack.org/octavia/latest/admin/index.html)
+* [Octavia configuration reference](https://docs.openstack.org/octavia/latest/configuration/index.html)
+
```
Comment (berendt) [inline docs/guides/configuration-guide/openstack/octavia.md:9]: Also add https://docs.openstack.org/octavia/latest/configuration/configref.html as Octavia configuration guide.

File: docs/guides/configuration-guide/openstack/manila.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+sidebar_label: Manila
+---
+
+# Manila
+
+* [Manila admin guide](https://docs.openstack.org/manila/latest/admin/index.html)
+* [Manila configuration reference](https://docs.openstack.org/manila/latest/configuration/index.html)
+
```
Comment (berendt) [inline docs/guides/configuration-guide/openstack/manila.md:9]: Also add https://docs.openstack.org/manila/latest/configuration/shared-file-systems/samples/manila.conf.html as Manila configuration reference.

**Pull request merged**